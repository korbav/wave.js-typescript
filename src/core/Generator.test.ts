import Generator from '../core/Generator';

describe('Generator', () => {
  it('should define the appropriate generators', () => {
    expect(Object.values(Generator)).toMatchSnapshot();
  });
});
