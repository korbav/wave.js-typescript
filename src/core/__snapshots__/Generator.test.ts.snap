// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generator should define the appropriate generators 1`] = `
Array [
  "bars",
  "bars blocks",
  "big bars",
  "cubes",
  "dualbars",
  "dualbars blocks",
  "fireworks",
  "flower",
  "flower blocks",
  "orbs",
  "ring",
  "rings",
  "round wave",
  "shockwave",
  "shine",
  "shine rings",
  "star",
  "static",
  "stitches",
  "wave",
  "web",
]
`;
